src/rules/hindi.toml

```diff
@@ -0,0 +1,19 @@
+min_trimmed_length = 0
```

Comment:
what is the shortest word that can form a sentence? That should probably be defined here.

Reply:
3 words. I'll update.